### ISSUE TITLE
Setup is unable to load isutils.dll

### ISSUE THREAD

**Issue by hydroculator** [doxygen/doxygen#8796]
When installing doxygen for Windows, using doxygen-1.9.2-setup.exe, I get the following two errors:

Setup is unable to load isutils.dll
Setup is unable to update PATH environment variable!

I have tried several times to install, as administrator and not.

I found this article from several years ago that describes this problem exactly, which attributes the issue to a misconfigured installation file. 

https://stackoverflow.com/questions/31372056/doxygen-unable-to-load-isutils-dll

I checked the .zip version of the distribution archive, but it doesn't contain the dll listed in the error message.

I also set the path manually to the bin folder created by the installer, but the installer still complains about not being able to set the path when I try to install again.

**Comment by albert-github** [comment]
A bit strange as I installed the 1.9.2 version on my Windows 10 system as well and had no problems, though I installed the 1.9.2 version and before the 1.9.1 version was present.

- Which version of Windows are you using?
- Are you installing doxygen over an existing version or is it new?

**Comment by albert-github** [comment]
As a workaround you could try to install an older release / older releases (they can be found at https://sourceforge.net/projects/doxygen/files/) and as last install 1.9.2 again.

**Comment by doxygen** [comment]
The dll should be part of the setup.exe and is only needed during installation (i.e. it is put in the temp dir and deleted afterwards). You can also download it here [isutils.dll.zip](https://github.com/doxygen/doxygen/files/7253940/isutils.dll.zip) to check if you can get it to work with it.

**Comment by albert-github** [comment]
@doxygen Which temp directory would this be? I looked during install in `%temp%` but here there are 3 directories created. 2 directories contain the file `doxygen-1.9.2-setup.tmp` of 2476KB and 1 directory contains a sub-directory `_issetup` with the files `_isdecmp.dll` and `_setup64.tmp`.

Note: on my system I can install doxygen (over an existing installation) and during the installation process no questions are asked about an installation directory.

**Comment by doxygen** [comment]
@albert-github I have this in my Innosetup config file:

```.ini
[Files]
; Defines the files to include, and which installation components they are to be included in
Source: "r:\isutils.dll";  DestDir: "{tmp}"; Flags: promptifolder  deleteafterinstall; Components: main; 
...
```
The `{tmp}` variable is documented here: 
https://jrsoftware.org/ishelp/index.php?topic=consts

The file exists at the source location:
```
dir r:\isutils.dll
 Volume in drive R is Shared Folders
 Volume Serial Number is 0100-0005

 Directory of r:\

12-01-2020  16:02            20.480 isutils.dll
               1 File(s)         20.480 bytes
               0 Dir(s)  622.584.307.712 bytes free
```

**Comment by albert-github** [comment]
@doxygen  
- my description in the https://github.com/doxygen/doxygen/issues/8796#issuecomment-930473524 was based on the fact that I canceled the installation at the moment that the installation should start. I now ran again doing an installation and during the install the file `isutils.dll` is create parallel to the sub-directory `_issetup`. So it looks like that this is OK.
- I looked at the `{tmp}` i.e. `C:\Windows\Temp` but I don't see a directory or file created here but this is probably due to the fact that I mapped the environment variable `tmp` and `temp` to the same directory (which is different from `C:\Windows\Temp`).
